bar/90_contents/UpdateEngine/updateVEvent.js

```diff
@@ -50,6 +50,10 @@ function(request){
     var info = personalBoxAccessor.getString(pathDavName);
     var accInfo = JSON.parse(info);
 
+    if (!(checkParams(request, params))){
+      return createResponse(400, {"error": "missing required parameter. " + JSON.stringify(params)})
```

Comment:
You are using params which are from the request. Should you specify which param is missing?

Reply:
Does that mean that the missing parameters are returned in response?